x/posts/internal/keeper/keeper_test.go

```diff
@@ -391,7 +391,7 @@ func TestKeeper_RemoveReaction(t *testing.T) {
 			expectedStored: types.Reactions{},
 		},
 		{
-			name:           "Non existing like returned expError - Creator",
+			name:           "Non existing like returned expError - Owner",
```

Comment:
Why has this changed?

```suggestion
			name:           "Non existing reaction returns error - Creator",
```